.changeset/little-hairs-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+`<rh-cta>`: made static icon size and positioned icon inline instead of flex
```

Comment:
changesets should describe user-facing improvements, not internal code changes. This prose could be a commit message, or a PR title, but shouldn't be in the changelog

Reply:
Could you also please update the PR title? I couldn't figure out which problem this is coming to address

Reply:
very clear. thanks! This is #238